api/stream/design.go

```diff
@@ -460,6 +476,27 @@ var SinkKeyRequest = func() {
 	Required("sinkId")
 }
 
+var StatisticsRequest = func() {
+	Attribute("since", String, func() {
+		Description("Starting date and time of the statistics.")
+		Format(FormatDateTime)
+		Example("2022-04-28T14:20:04.000Z")
+	})
+	Required("since")
+	Attribute("until", String, func() {
+		Description("Ending date and time of the statistics.")
+		Format(FormatDateTime)
+		Example("2022-04-28T14:20:04.000Z")
+	})
+	Required("until")
+	Attribute("duration", Int64, func() {
+		Description("Duration of each interval in milliseconds.")
+		Example(123456789)
+		Minimum(int64(time.Hour))
+	})
+	Required("duration")
```

Comment:
`duration` -> `interval`? `oneInterval` ? or `intervalDuration`?

It can be misinterpreted as `since + duration = until`.